gem_output.json

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
Can we move `gem_output.json` and `integration_versions.md` to the `docs` folder?

Reply:
Yeah we just need to be cognizant of not pinging the docs team as a reviewer (yet) as we want to ensure that the language we are presenting is consistent with what we and product want.
Looking at CODEOWNERS we should be able to move these to `docs` without pinging them, so that sounds good to me.